app/src/main/java/at/vunfer/openrealms/MainActivity.java

```diff
@@ -273,14 +281,18 @@ public void run() {
                                     overlayViewPresenter.updateTurnHealing(stats.getTurnHealing());
                                     overlayViewPresenter.updateTurnCoin(stats.getTurnCoin());
                                     if (stats.getPlayerHealth() < 1) {
-                                        ImageView victoryImage = findViewById(R.id.victory_image);
+                                        victoryImage = findViewById(R.id.victory_image);
                                         victoryImage.setVisibility(View.VISIBLE);
                                         victoryImage.getParent().bringChildToFront(victoryImage);
-                                        Button endTurnButton = findViewById(R.id.end_turn_button);
-                                        endTurnButton.setVisibility(View.GONE);
+                                        endTurnButton = findViewById(R.id.end_turn_button);
+                                        endTurnButton.setVisibility(View.INVISIBLE);
                                         for (CardView c : cardViews) {
                                             c.setFaceDown();
                                         }
+                                        defeatImage = findViewById(R.id.defeat_image);
+                                        defeatImage.setVisibility(View.INVISIBLE);
+                                        endTurnButton = findViewById(R.id.end_turn_button);
+                                        endTurnButton.setVisibility(View.INVISIBLE);
```

Comment:
I think there was a merge conflict here, I doubt that we need to find and set the button to Invisible twice.

Reply:
Must have been it yea! Should be good now.